pds4indextools/pds4_create_xml_index.py

```diff
@@ -434,26 +532,50 @@ def main():
         namespaces['pds'] = namespaces.pop(None)
         prefixes = {v: k for k, v in namespaces.items()}
 
-        xml_results = {}
-        traverse_and_store(root, tree, xml_results, elements_to_scrape,
+        label_results = {}
+        traverse_and_store(root, tree, label_results, elements_to_scrape,
                            nillable_elements_info, config, file)
 
-        for key in list(xml_results.keys()):
-            process_tags(xml_results, key, root,
-                         namespaces, prefixes, args)
+        for key in list(label_results.keys()):
```

Comment:
No reason to use list() here

Reply:
Okay I actually came back to this because I didn't test it correctly. `list()` is actually needed here, since otherwise the dictionary is changed in-place. I get this error if the `list` is removed:

```
Traceback (most recent call last):
  File "/Users/emiliesimpson/Github/rms-pds4indextools/pds4indextools/pds4_create_xml_index.py", line 630, in <module>
    main()
  File "/Users/emiliesimpson/Github/rms-pds4indextools/pds4indextools/pds4_create_xml_index.py", line 579, in main
    for key in label_results.keys():
RuntimeError: dictionary changed size during iteration
```